searchMatrix2.cc

```diff
@@ -0,0 +1,27 @@
+// Time Complexity : O(n)
+// Space Complexity : O(1)
+// Did this code successfully run on Leetcode : It ran successfully
+// Any problem you faced while coding this : None
```

Comment:
Time complexity will be O(m+n), where m and n are rows and columns respectively.